std/range/package.d

```diff
@@ -7420,16 +7420,23 @@ and the element member is named $(D value).
 
 The index starts at $(D start) and is incremented by one on every iteration.
 
-Bidirectionality is propagated only if $(D range) has length.
-
 Overflow:
-If $(D range) has length, then it is an error to pass a value for $(D start)
-so that $(D start + range.length) is bigger than $(D Enumerator.max), thus it is
-ensured that overflow cannot happen.
+    If $(D range) has length, then it is an error to pass a value for $(D start)
+    so that $(D start + range.length) is bigger than $(D Enumerator.max), thus
+    it is ensured that overflow cannot happen.
+
+    If $(D range) does not have length, and $(D popFront) is called when
+    $(D front.index == Enumerator.max), the index will overflow and
+    continue from $(D Enumerator.min).
```

Comment:
I prefer backticks, because they look nicer, but feel free to leave `$D(..)` as it is.